picard/acousticbrainz/__init__.py

```diff
@@ -0,0 +1,121 @@
+# -*- coding: utf-8 -*-
+#
+# Picard, the next-generation MusicBrainz tagger
+#
+# Copyright 2014 Music Technology Group - Universitat Pompeu Fabra
+# Copyright 2020-2021 Gabriel Ferreira (@gabrielcarvfer)
+#
+# This program is free software; you can redistribute it and/or
+# modify it under the terms of the GNU General Public License
+# as published by the Free Software Foundation; either version 3
+# of the License, or (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program; if not, write to the Free Software
+# Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301, USA.
+
+
+from functools import partial
+import json
+import os
+import subprocess
+import sys
+
+from picard.acousticbrainz.extractor import (
+    check_extractor_version,
+    precompute_extractor_sha,
+)
+from picard.util import load_json
+from picard.webservice import ratecontrol
+
+
+ACOUSTICBRAINZ_HOST = "acousticbrainz.org"
+ACOUSTICBRAINZ_PORT = 80
+
+ratecontrol.set_minimum_delay((ACOUSTICBRAINZ_HOST, ACOUSTICBRAINZ_PORT), 1000)
+
+acousticbrainz_extractor = os.path.dirname(sys.argv[0]) + os.sep + "streaming_extractor_music" + ("" if sys.platform != "win32" else ".exe")

```

Comment:
You can use `picard.util.find_executable` here.

Reply:
Done.